wmn-data.json

```diff
@@ -2311,7 +2311,7 @@
         "uri_check" : "https://gitlab.com/{account}",
         "e_code" : 200,
         "e_string" : "Member since",
-        "m_string" : "GitLab.com offers free unlimited",
+        "m_string" : "You are being <a href="https://gitlab.com/users/sign_in">redirected</a>",

```

Comment:
Looks like you have an error in the JSON here with an unescaped double quote `"`. Can you change the `m_string` to not have a double quote in it?

Reply:
ah, how stupid :) Fixed with cfdc055db0f55d8c6f34ef27a0545d63523132bf